Src/FluentAssertions/Equivalency/DataColumnEquivalencyStep.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+
+using FluentAssertions.Data;
+using FluentAssertions.Execution;
+
+namespace FluentAssertions.Equivalency
+{
+    public class DataColumnEquivalencyStep : IEquivalencyStep
+    {
+        /// <summary>
```

Comment:
🔧 You don't have to repeat the documentation you inherited from the interface.

Reply:
Okay, I was just copying what I saw in another randomly-picked equivalency step implementation. :-)